docs/process/releasing-updates.md

```diff
@@ -75,18 +104,47 @@ Here's an example of the previous changelog draft afterit has been edited:
 }
 ```
 
-Once you are happy with those changes, commit and push them to GitHub. Get the others on the team to :thumbsup: them if you're not sure.
+Once you've pushed up the version update and the changelog changes, you're ready to release! Get the others on the team to :thumbsup: in a PR if you're not sure. Note that any version change that does not have an associated changelog entry will not successfully release.
+
+### 3. Releasing
 
 When you feel ready to start the deployment, run this command in Chat:
 
 ```
 .release! desktop/YOUR_BRANCH to {production|beta|test}
 ```
 
+If you are releasing from master, YOUR_BRANCH is unnecessary; write:
+```
+.release! desktop to {production|beta|test}
+``` 
+
 We're using `.release` with a bang so that we don't have to wait for any current CI on the branch to finish. This might feel a little wrong, but it's OK since making the release itself will also run CI.
 
-If you're releasing a production update, release a beta update for the next version too, so that beta users are on the latest release.
+If you're releasing a `production` update, release a `beta` update for the next version too, so that beta users are on the latest release.
+
+### 4. Check for Completed Release
 
-## Error Reporting
+Go to [Central's Deployments](https://central.github.com/deployments) to find your release; you'll see something at the top of the page like:
+```
+desktop/desktop deployed from {YOUR_BRANCH}@{HASH_ABBREVIATION_FOR_COMMIT} to {production|beta|test}
+```
+it will initially specify its state as `State: pending` and will be completed when it says `State: released`
+
+You will also see this in Chat:
+`desktopbot tagged desktop/release-{YOUR_VERSION}`
+  
+When it's in `State: released` for `beta` or `production`, switch to your desktop application and make sure that the corresponding (prod|beta) app auto-updates.
+If you don't have the app for `beta`, for example, you can always download the previous version on Central to see it update (but make sure you move it out of the Downloads folder and into the Applications folder for macOS or it won't auto-update).
```

Comment:
:art: Maybe some extra words here about how important testing the updater is important:

> Testing an update is detected, downloaded and applied correctly is very important - if this is somehow broken during development then our users will not likely stay up to date!